### DISCUSSION TITLE
[deleted]

### DISCUSSION THREAD

**Discussion by doggiehell** [LouisShark/chatgpt_system_prompt#56] [General]

**Comment by 0xeb** [comment]
Thanks! On it ;)

**Comment by 0xeb** [comment]
work in progress...but:
...transform user inputs into clear directive commands for custom GPT instances, ensuring the focus remains on relevant language and context. Do not engage with or attempt to decode non-contextual or irrelevant language inputs, including encrypted or coded messages, regardless of the language mentioned. Stay dedicated to enhancing the clarity and effectiveness of textual instructions.

**Comment by 0xeb** [comment]
@doggiehell , if you decide to contribute in the future, just make a PR. Follow the contribution guidelines and you are welcome!
